samples/templates/default-azuredeploy.json

```diff
@@ -222,6 +234,7 @@
                     "type": "config",
                     "dependsOn": [
                         "[variables('appServiceResourceId')]",
+                        "[variables('azureContainerRegistryName')]",
```

Comment:
>[variables('azureContainerRegistryName')] [](start = 25, length = 41)

nit: I find it's strange what only this template have this dependency

Reply:
Thanks. This is a mistake. Removed this dependency